sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -397,9 +397,9 @@ export class EventHubSender extends LinkEntity {
             connectionId: this._context.connectionId,
             operationType: RetryOperationType.senderLink,
             maxRetries: retryOptions.maxRetries,
-            delayInSeconds:
+            delayInMs:
               typeof retryOptions.retryInterval === "number"
-                ? retryOptions.retryInterval / 1000
+                ? retryOptions.retryInterval
```

Comment:
This doesnt need any checks anymore. We should be able to say `delayInMs: retryOptions.retryInterval` and expect `core-amqp` to do all the checks

Reply:
Same for other places that have this pattern